src/util/loginAPI.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+//import User  from './Api';
+
+axios.defaults.headers.post['Content-Type'] = 'application/json';
+axios.defaults.headers.put['Content-Type'] = 'application/json';
+axios.defaults.baseURL =
+  process.env.NODE_ENV === 'development' ? 'http://localhost:4565/api' : '/csi/api';
+
+class API {
+  static async login(nid: string, password: string): Promise<string> {
```

Comment:
The login API returns a response that will eventually have this structure:
```json
{
    "token": "some string",
    "nid": "some nid",
    "ID": 1,
    "email": "some.email@example.com"
}
```
So this Promise should have a different type. Check out the client code for an example: 

https://github.com/CHDR-Inventory-System/client/blob/feature/dashboard/src/types/API.d.ts#L13-L19